actpool/actqueue.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 type nonceWithTTL struct {
+	idx      int
```

Comment:
I add a `idx` property, and it is only use for benchmark `heap.Remove`, is it acceptable?

Reply:
it's ok